library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```
@@ -79,6 +79,21 @@
   }
 }
 
+@media (min-width: 767px) {
+  .sidebar-site-title {
+    width: 128px;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    line-clamp: 2;
+    -webkit-box-orient: vertical;
```

I think all -webkit- props should be there without as well: 
```suggestion
    display: -webkit-box;
    -webkit-line-clamp: 2;
    -webkit-box-orient: vertical;
    display: box;
    line-clamp: 2;
    box-orient: vertical;
```
Could you test that?

---

This should work. But can you explain,  why is this better?

---

As I understand it, at the moment only the webkit prefixed properties are supported by the browsers, later after the "normal" ones are supported by the browsers, support for the prefixed properties will be discontinued. This is when those take over.

---

box-orient is deprecated, so we shouldn't use that.

https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient

---

And, reading on caniuse, it seems that the -webkit prefixed box-orient is deprecated. It's probably a bad idea to depend on those rules. If they remove support in a new version it will just break for users.